src/test/suite/playground.test.ts

```diff
@@ -97,64 +97,64 @@ suite('Playground', function () {
     await vscode.commands.executeCommand('mdb.createPlayground');
 
     const editor = vscode.window.activeTextEditor;
-    if (!editor) {
-      throw new Error('Window active text editor is undefined');
+    expect(editor).to.exist;
+
+    if (editor) {
```

Comment:
nit: Why are we wrapping the rest of the code in an `if (editor)` conditional if we just asserted that `editor` is not null-ish? I’m guessing it’s related to typescript not understanding chai assertions as type assertions, but the previous code seemed like exactly the right thing to do in that case.